lib/src/interfaces.dart

```diff
@@ -16,64 +16,63 @@ part of pdfjs;
 
 class AnnotationLayerBuilderOptions {
   JsObject _jsInternal;
-  IL10n _l10n;
-  PDFPageProxy _pdfPage;
+  IL10n? _l10n;
+  PDFPageProxy? _pdfPage;
 
-  AnnotationLayerBuilderOptions() {
+  AnnotationLayerBuilderOptions() :
     _jsInternal = JsObject.jsify({});
-  }
 
-  IL10n get l10n => _l10n;
-  set l10n(IL10n l10n) {
+  IL10n? get l10n => _l10n;
+  set l10n(IL10n? l10n) {
     _jsInternal['l10n'] = l10n;
 
     _l10n = l10n;
   }
 
-  DivElement get pageDiv => _jsInternal['pageDiv'] as DivElement;
-  set pageDiv(DivElement pageDiv) {
+  DivElement? get pageDiv => _jsInternal['pageDiv'] as DivElement?;
+  set pageDiv(DivElement? pageDiv) {
     _jsInternal['pageDiv'] = pageDiv;
   }
 
-  PDFPageProxy get pdfPage => _pdfPage;
-  set pdfPage(PDFPageProxy pdfPage) {
-    _jsInternal['pdfPage'] = pdfPage._jsInternal;
+  PDFPageProxy? get pdfPage => _pdfPage;
+  set pdfPage(PDFPageProxy? pdfPage) {
+    _jsInternal['pdfPage'] = pdfPage?._jsInternal;
 
     _pdfPage = pdfPage;
   }
 
-  bool get renderInteractiveForms => _jsInternal['renderInteractiveForms'] as bool;
-  set renderInteractiveForms(bool renderInteractiveForms) {
+  bool? get renderInteractiveForms => _jsInternal['renderInteractiveForms'] as bool?;
+  set renderInteractiveForms(bool? renderInteractiveForms) {
     _jsInternal['renderInteractiveForms'] = renderInteractiveForms;
   }
 }
 
 enum AnnotationLayerBuilderIntent { display }
 
 class AnnotationLayerBuilder {
-  JsObject _jsInternal;
 
-  AnnotationLayerBuilder(AnnotationLayerBuilderOptions options) {
-    _jsInternal = JsObject(PDFJS.versionSafePdfJsViewerContext['AnnotationLayerBuilder'] as JsFunction, [
+  AnnotationLayerBuilder(AnnotationLayerBuilderOptions options) :
```

Comment:
Not specifically used by w_viewer, though an area of note, as we typically see our support issues arise surrounding annotations. Would be good to test pdfs with different annotations